lisa/base_tools/rpm.py

```diff
@@ -31,8 +31,25 @@ def get_file_size(self, file: str) -> int:
 
     def install_local_package(self, file: str) -> None:
         self.run(
-            f"-ivh {file}",
+            f'-ivh "{file}"',
             sudo=True,
             expected_exit_code=0,
             expected_exit_code_failure_message=(f"failed to install {file}"),
         )
+
+    def install_local_packages_in_dir(
```

Comment:
Let's use the install single file method. It may need to select which packages install. So we can control this list in transformer.

Reply:
The input given to `install_local_packages_in_dir` is a remote directory which is created by the transformer. The rpms are copied into this directory by the transformer itself. So, control of the list is with the transformer already.

Reply:
Will remove this function. As discussed in the other comment, this wont be needing.